From 9180cc22b99c5da120c979e347397eb91c718ee4 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 29 Jun 2021 14:16:24 +0200
Subject: [PATCH] imx8mm-var-dart-plt: Switch dtb name for kernel 5.4.3

We have switched to kernel 5.4.3 which uses a different dtb name so
let's use the new name here.

PLT board uses kernel 5.4.3 and the NRT one uses 4.19 with
RT patches. Starting with kernel 5.4.85 the dtb names were changed
to account for the old and new devkits.

Upstream-status: Inappropriate[configuration]
Signed-off-by: Florin Sarbu <florin@balena.io>
---
 include/configs/imx8mm_var_dart.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 15da1ec940..2c1efc1b50 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -124,11 +124,7 @@
 					"setenv fdt_file imx8mm-var-som-symphony.dtb; " \
 				"fi; " \
 			"else " \
-				"if test $carrier_rev = legacy; then " \
-					"setenv fdt_file imx8mm-var-dart-dt8mcustomboard-legacy.dtb; " \
-				"else " \
-					"setenv fdt_file imx8mm-var-dart-dt8mcustomboard.dtb; " \
-				"fi; " \
+				"setenv fdt_file imx8mm-var-dart.dtb; " \
 			"fi; " \
 		"fi; \0" \
 	"loadfdt=run findfdt; " \
-- 
2.17.1

